### ISSUE TITLE
Retire wiki.bx.psu.edu and port out relevant pages somewhere

### ISSUE THREAD

**Issue by pashadag** [psu-ccbb/ccbb-site#11]
Most of the stuff on the old wiki is outdated, but there are still some pages that may still be relevant, e.g. with creating accounts/printing/cluster/etc. We should decide on a place (github? or under the new CCBB page?) and format (as is? makrdown?) that the new stuff can be hosted under.

**Comment by richard-burhans** [comment]
I agree.  The CCBB page is currently hosted on github.  The current wiki uses an old version of MediaWiki.  I'm sure someone has written some code we could use to convert the MediaWiki pages to GitHub flavored markdown.  Maybe this could be a project for Cathy.
